src/models/Course.js

```diff
@@ -2,6 +2,16 @@ module.exports = (sequelize, DataTypes) => {
   const obj = sequelize.define('course', {
     name: DataTypes.STRING,
     shortcode: DataTypes.STRING,
+    isUnlisted: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
+    },
+    questionFeedback: {
```

Comment:
Is there a reason this is associated with a course as a whole instead of a specific queue? I could foresee a case where a course might want feedback enabled on one of its queues but not on others.

Reply:
Hmm... I think that's a valid point. But, say there's a user with multiple queues who just don't want the questionFeedback at all - they'd probably be annoyed if they had to turn off questionFeedback for each one of those. @wadefagen thoughts?

Reply:
So, even though it defaults to true here, on the client, the toggle defaults to false. So in practice, this will be disabled for all new courses to begin with (which I think is actually a good thing, based in how many people have asked "can I disabled this?"). I think most people and most queues will want this off - I view this as a pretty special case, even within a course.

Reply:
Yeah, I feel like most people just want this gone ASAP -- I'm pretty sure #289 and probably other issues are open about this whole questionFeedback thing